package.json

```diff
@@ -66,7 +66,7 @@
   "scripts": {
     "serve": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --coverage",
```

Comment:
Why was the coverage flag removed here?

Reply:
@Rumone because it was not there, I added it for my convenience on previous commit

Reply:
Oh okay